internal/driver/helper.go

```diff
@@ -153,30 +187,11 @@ func randomFloat(min, max float64) float64 {
 	}
 }
 
-func parseStrToFloat(str string, bitSize int) (float64, error) {
-	if f, err := strconv.ParseFloat(str, bitSize); err != nil {
-		return f, err
-	} else {
-		return f, nil
-	}
-}
-
-func parseFloatMinimumMaximum(minimum, maximum, dataType string) (float64, float64, error) {
-	var err, err1, err2 error
-	var min, max float64
-	switch dataType {
-	case common.ValueTypeFloat32, common.ValueTypeFloat32Array:
-		min, err1 = parseStrToFloat(minimum, 32)
-		max, err2 = parseStrToFloat(maximum, 32)
-		if max <= min || err1 != nil || err2 != nil {
-			err = fmt.Errorf("minimum:%s maximum:%s not in valid range, use default value", minimum, maximum)
-		}
-	case common.ValueTypeFloat64, common.ValueTypeFloat64Array:
-		min, err1 = parseStrToFloat(minimum, 64)
-		max, err2 = parseStrToFloat(maximum, 64)
-		if max <= min || err1 != nil || err2 != nil {
-			err = fmt.Errorf("minimum:%s maximum:%s not in valid range, use default value", minimum, maximum)
-		}
+func isValid(minimum, maximum *float64) bool {
+	valid := true
+	if minimum != nil && maximum != nil &&
+		((*maximum < *minimum) || (*minimum == 0 && *maximum == 0)) {
```

Comment:
(*minimum == 0 && *maximum == 0) should be a valid condition

Reply:
Updated, allow min equal to max.